docs/source/en/model_doc/mllama.md

```diff
@@ -14,93 +14,110 @@ rendered properly in your Markdown viewer.
+# MLlama
 
+[MLlama](https://huggingface.co/papers/2407.21783), or Llama 3.2 Vision, is a multimodal version of [Llama 3](./llama3), available in 11B and 90B parameters, and as an instruction-tuned and base variant. This model integrates a separately trained vision adapter (image encoder, image adapter, and video adapter) to handle visual tasks like image reasoning, captioning, and answering. The instruction-tuned variants are post-trained with supervised fine-tuning (SFT) and reinforcement learning with human feedback (RLHF) to align with human helpfulness and safety values.
 
-<Tip warning={true}>
+You can find all the original MLlama checkpoints under the [Llama 3.2](https://huggingface.co/collections/meta-llama/llama-32-66f448ffc8c32f949b04c8cf) collection.
 
-Mllama has an extra token used as a placeholder for image positions in the text. It means that input ids and an input embedding layer will have an extra token. But since the weights for input and output embeddings are not tied, the `lm_head` layer has one less token and will fail if you want to calculate loss on image tokens or apply some logit processors. In case you are training, make sure to mask out special `"<|image|>"` tokens in the `labels` as the model should not be trained on predicting them.
+> [!TIP]
+> Click on the MLlama models in the right sidebar for more examples of how to apply MLlama to different vision-language tasks like image captioning, visual question answering, and reasoning.
 
-Otherwise if you see CUDA-side index erros when generating, use the below code to expand the `lm_head` by one more token. 
+The example below demonstrates how to generate text based on an image with [`Pipeline`] or the [`AutoModel`] class.
 
+<hfoptions id="usage">
+<hfoption id="Pipeline">
 
 ```python
-old_embeddings = model.get_output_embeddings()
-
-num_tokens = model.vocab_size + 1
-resized_embeddings = model._get_resized_lm_head(old_embeddings, new_num_tokens=num_tokens, mean_resizing=True)
-resized_embeddings.requires_grad_(old_embeddings.weight.requires_grad)
-model.set_output_embeddings(resized_embeddings)
+import torch
+from transformers import pipeline
+
+pipeline = pipeline(
+    task="image-text-to-text",
+    model="meta-llama/Llama-3.2-11B-Vision-Instruct",
+    device=0,
+    torch_dtype=torch.bfloat16
+)
+messages = [
+    [
+        {
+            "role": "user", 
+            "content": [
+                {"type": "image", "url": "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/pipeline-cat-chonk.jpeg"},
+                {"type": "text", "text": "What does the image show?"}
+            ]
+        }
+    ],
+]
+pipeline(text=messages, return_full_text=False)
 ```
-</Tip>
-
 
-## Usage Example
+</hfoption>
+<hfoption id="AutoModel">
 
-#### Instruct model
 ```python
 import torch
-from transformers import MllamaForConditionalGeneration, AutoProcessor
+from transformers import BitsAndBytesConfig, MllamaForConditionalGeneration, AutoProcessor
```

Comment:
```suggestion
from transformers import MllamaForConditionalGeneration, AutoProcessor
```

Reply:
The `AutoModel` example shouldn't show quantization usage, so it was fine the way it was before. I was just removing `BitsAndBytesConfig` from the import